2010-10-04  Michael Tautschnig  <mt@debian.org>

	* examples/simple/disk_config/FAIBASE: Added bootable:1 as requested by
		Maximiliano Curia (apparently still necessary on some systems).
	* conf/{grub.cfg,menu.lst},examples/simple/files/etc/fai/menu.lst/FAISERVER:
		Change hostname= to HOSTNAME= for kernel command line (thanks Karl Wallner
		for testing).
	* examples/simple/package_config/GNOME: Remove gnome-apt, ice*-gnome-support
		as these don't exist in squeeze anymore (thanks Karl Wallner for reporting
		this)
Index: trunk/conf/grub.cfg
===================================================================
--- trunk.orig/conf/grub.cfg
+++ trunk/conf/grub.cfg	
@@ -48,25 +48,25 @@
 
 menuentry "Fully Automatic Installation - demohost (pw: installme)" {
     set gfxpayload=1024x768
-    linux   /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.1.1:::::eth0:off hostname=demohost
+    linux   /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.1.1:::::eth0:off HOSTNAME=demohost
     initrd  /boot/initrd.img
 }
 
 menuentry "Fully Automatic Installation with GNOME (pw: installme)" {
     set gfxpayload=1024x768
-    linux /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.1.1:::::eth0:off hostname=gnomehost
+    linux /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.1.1:::::eth0:off HOSTNAME=gnomehost
     initrd /boot/initrd.img
 }
 
 menuentry "Fully Automatic Installation - faiserver (pw: installme)" {
     set gfxpayload=1024x768
-    linux /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.1.250::192.168.1.254:255.255.255.0::xxx:off hostname=faiserver
+    linux /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.1.250::192.168.1.254:255.255.255.0::xxx:off HOSTNAME=faiserver
     initrd /boot/initrd.img
 }
 
 menuentry "FAI rescue system, no installation" {
     set gfxpayload=1024x768
-    linux /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=sysinfo ip=192.168.1.1:::::eth0:off hostname=demohost
+    linux /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=sysinfo ip=192.168.1.1:::::eth0:off HOSTNAME=demohost
     initrd /boot/initrd.img
 }
 
Index: trunk/conf/menu.lst
===================================================================
--- trunk.orig/conf/menu.lst
+++ trunk/conf/menu.lst	
@@ -26,19 +26,19 @@
 
 title Fully Automatic Installation - demohost (pw: installme)
 password installme
-kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.33.1:::::eth0:off hostname=demohost
+kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.33.1:::::eth0:off HOSTNAME=demohost
 initrd /boot/initrd.img
 
 title Fully Automatic Installation with GNOME (pw: installme)
 password installme
-kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.33.1:::::eth0:off hostname=gnomehost
+kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.33.1:::::eth0:off HOSTNAME=gnomehost
 initrd /boot/initrd.img
 
 title Fully Automatic Installation - faiserver (pw: installme)
 password installme
-kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.33.250::192.168.33.254:255.255.255.0::xxx:off hostname=faiserver
+kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.33.250::192.168.33.254:255.255.255.0::xxx:off HOSTNAME=faiserver
 initrd /boot/initrd.img
 
 title FAI rescue system, no installation
-kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=sysinfo ip=192.168.33.1:::::eth0:off hostname=demohost
+kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=sysinfo ip=192.168.33.1:::::eth0:off HOSTNAME=demohost
 initrd /boot/initrd.img
Index: trunk/examples/simple/disk_config/FAIBASE
===================================================================
--- trunk.orig/examples/simple/disk_config/FAIBASE
+++ trunk/examples/simple/disk_config/FAIBASE	
@@ -2,7 +2,7 @@
 #
 # <type> <mountpoint> <size>   <fs type> <mount options> <misc options>
 
-disk_config disk1 disklabel:msdos fstabkey:uuid
+disk_config disk1 disklabel:msdos bootable:1 fstabkey:uuid
 
 primary /      250      ext3  rw,noatime,errors=remount-ro
 logical swap   200-1000 swap  rw                   
Index: trunk/examples/simple/files/etc/fai/menu.lst/FAISERVER
===================================================================
--- trunk.orig/examples/simple/files/etc/fai/menu.lst/FAISERVER
+++ trunk/examples/simple/files/etc/fai/menu.lst/FAISERVER	
@@ -26,19 +26,19 @@
 
 title Fully Automatic Installation - demohost (pw: installme)
 password installme
-kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.33.1:::::eth0:off hostname=demohost
+kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.33.1:::::eth0:off HOSTNAME=demohost
 initrd /boot/initrd.img
 
 title Fully Automatic Installation with GNOME (pw: installme)
 password installme
-kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.33.1:::::eth0:off hostname=gnomehost
+kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.33.1:::::eth0:off HOSTNAME=gnomehost
 initrd /boot/initrd.img
 
 title Fully Automatic Installation - faiserver (pw: installme)
 password installme
-kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.33.250::192.168.33.254:255.255.255.0::xxx:off hostname=faiserver
+kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=install ip=192.168.33.250::192.168.33.254:255.255.255.0::xxx:off HOSTNAME=faiserver
 initrd /boot/initrd.img
 
 title FAI rescue system, no installation
-kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=sysinfo ip=192.168.33.1:::::eth0:off hostname=demohost
+kernel /boot/vmlinuz boot=live FAI_FLAGS="verbose,createvt" FAI_ACTION=sysinfo ip=192.168.33.1:::::eth0:off HOSTNAME=demohost
 initrd /boot/initrd.img
Index: trunk/examples/simple/package_config/GNOME
===================================================================
--- trunk.orig/examples/simple/package_config/GNOME
+++ trunk/examples/simple/package_config/GNOME	
@@ -2,10 +2,7 @@
 
 iceweasel
 icedove
-iceweasel-gnome-support
-icedove-gnome-support
 menu gdm
-gnome-apt
 gnome-core
 desktop-base
 evince gconf-editor
